### PR TITLE
Semaphore CI setup

### PR DESCRIPTION
**Pull request by garyluu**
For my personal use and those who wish to fork repo and test separate from Dockstore.  Shouldn't affect the rest of the team.

Also attempts ga4gh/dockstore#2145 (a better fix would be implement progress bar when switching between tools/workflows and have Cypress check that there's no progress bar before proceeding with other actions)

### CODE REVIEW

File: .semaphore/semaphore.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  - name: "Build"
+    task:
+      env_vars:
+        - name: APP_ENV
+          value: prod
+        - name: WEBSERVICE_VERSION
+          value: 1.6.0-alpha.6
+      prologue:
+        commands:
+          - nvm use 10.10.0
```
Comment (coverbeck) [inline .semaphore/semaphore.yml]: Just curious, why is this different than .travis.yml (10.13.0)?

File: .semaphore/semaphore.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  - name: "Integration Tests"
+    task:
+      env_vars:
+        - name: WEBSERVICE_VERSION
+          value: 1.6.0-alpha.6
+      secrets:
+        - name: cypress
+      prologue:
+        commands:
+          - nvm use 10.10.0
```
Comment (coverbeck) [inline .semaphore/semaphore.yml]: Same as previous comment

File: .semaphore/semaphore.yml (hunk 1 above)
Comment (garyluu) [reply .semaphore/semaphore.yml]: Closest version that Semaphore has

File: .semaphore/semaphore.yml (hunk 2 above)
Comment (garyluu) [reply .semaphore/semaphore.yml]: Same as previous comment

File: .semaphore/semaphore.yml (hunk 1 above)
Comment (garyluu) [reply .semaphore/semaphore.yml]: Nevermind, was reading old docs https://semaphoreci.com/docs/languages/javascript/javascript-support-on-semaphore.html vs https://docs.semaphoreci.com/article/82-language-javascript-and-nodejs

**Pull request merged**